api/src/main/java/com/epam/pipeline/manager/preference/PreferenceValidators.java

```diff
@@ -145,6 +145,14 @@
         return (pref, dependencies) -> StringUtils.isNumeric(pref) && Long.parseLong(pref) > x;
     }
 
+    public static BiPredicate<String, Map<String, Preference>> isGreaterThanValueOrNull(String key) {
+        return (pref, dependencies) -> {
+            Long valueToCompare = dependencies.containsKey(key) ? dependencies.get(key).get(Long::parseLong) : Long.MIN_VALUE;
+            return StringUtils.isBlank(pref) ||
+                    StringUtils.isNumeric(pref) && Long.parseLong(pref) > valueToCompare;
```

Comment:
`pref value` shall be <= `valueToCompare`

Reply:
@mzueva did you mean `>=`?